ProjectSettings/ProjectSettings.asset

```diff
@@ -862,8 +864,9 @@ PlayerSettings:
   webGLPowerPreference: 2
   scriptingDefineSymbols:
     Android: TILT_BRUSH
-    Standalone: USD_SUPPORTED;UNITY_HAS_GOOGLEVR;TILT_BRUSH;FBX_SUPPORTED;FBXSDK_RUNTIME;LATK_SUPPORTED
-    Windows Store Apps: USD_SUPPORTED;UNITY_HAS_GOOGLEVR;TILT_BRUSH
+    Standalone: TILT_BRUSH;USD_SUPPORTED;FBX_SUPPORTED;FBXSDK_RUNTIME;LATK_SUPPORTED
```

Comment:
Did you mean to remove things from `Standalone`?

Reply:
Yes, `UNITY_HAS_GOOGLEVR` is obsolete, especially as we removed the gvr code.